Gems/Atom/RHI/Code/Include/Atom/RHI.Reflect/AllocatorManager.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/base.h>
+#include <AzCore/Memory/AllocationRecords.h>
+#include <AzCore/std/algorithm.h>
+#include <AzCore/std/parallel/mutex.h>
+#include <AzCore/std/string/string.h>
+
+namespace AZ::RHI
+{
+    class AllocatorManager
+    {
+    public:
+
+        AllocatorManager();
```

Comment:
Can we not use AZ::AllocatorManager for our needs instead of duplicating the class within RHI?

Reply:
yeah that's what I figured. What are your thoughts on renaming them to be called PassThroughAllocatorManager/PassThroughAllocators